nautobot/dcim/tables/template_code.py

```diff
@@ -62,9 +62,51 @@
 """
 
 INTERFACE_IPADDRESSES = """
-{% for ip in record.ip_addresses.all %}
-    <a href="{{ ip.get_absolute_url }}">{{ ip }}</a><br />
-{% endfor %}
+{% if record.interface.ip_addresses %}
```

Comment:
Do we need the if statement here? It will be a no-op even without it right?

Reply:
thanks - I removed the conditionals from the new template code snippets.